Geometry/TrackerCommonData/data/PhaseI/R34F16/pixfwd.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0"?>
+<DDDefinition xmlns="http://www.cern.ch/cms/DDL" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.cern.ch/cms/DDL ../../../DetectorDescription/Schema/DDLSchema.xsd">
+    
+
+<ConstantsSection label="pixfwd.xml" eval="true">
+ <Constant name="RootRadius"     value="[pixfwdCylinder:CylindersOuterRmax]"/>
+ <Constant name="RootHalfLength" value="[pixfwdCylinder:CylindersOuterLength]/2.+ [pixfwdCylinder:CylindersEndFlangeLength]/2."/>
+
+ <Constant name="AnchorZ"        value="0.*mm"/>
+ <Constant name="ZPixelForward"  value="291.*mm"/>
+ <Constant name="RootStartZ"     value="-4*mm"/> <!-- -2*mm-->
+ <Constant name="RootMidZ1"      value="[cms:TrackLumiZ1]-[ZPixelForward]"/>
+ <Constant name="RootMidZ2"      value="[cms:TrackLumiZ2]-[ZPixelForward]"/>
+ <Constant name="RootMidZ3"      value="[cms:TrackBeamZ1]-[ZPixelForward]"/>
+ <Constant name="RootEndZ"       value="2*[RootHalfLength]"/>
+
+ <Constant name="Disk1Z"         value="[pixfwdDisks:DiskHalfWidth]+[RootStartZ]"/> <!-- 35.75*mm-->
+ <Constant name="Disk2Z"         value="[Disk1Z]+75.0*mm"/> <!-- +72.5*mm--> <!-- +105.0*mm -->
+ <Constant name="Disk3Z"         value="[Disk2Z]+95.0*mm"/> <!-- +120.0*mm-->
+
+ <Constant name="ZCylinder"      value="[AnchorZ]"/>
+
+</ConstantsSection>
+
+<SolidSection label="pixfwd.xml">
+ <Polycone name="PixelForward" startPhi="0*deg" deltaPhi="360*deg" >
+ <ZSection z="[RootStartZ]"  rMin="[cms:TrackBeamR1]" rMax="[RootRadius]"/>
```

Comment:
@schneiml - I have not checked this scenario, however, I do not see appropriate changes in this file.